fr-portfolio.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+</head>
+<body>	
+	<header class="navbar">
+		<a href="#portfolio">Portfolio</a>
+		<a href="#about">About</a>
+		<a href="#contact">Contact</a>
+	</header>
+	<section class="welcome-banner" >
+							<h1>Freddie's Portfolio</h1>
+										
+	</section>	
+	<section id="about" class="about-alt">
+		<h2> About me </h2>
+			<div class="about-me">
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+				</div>
+				<div>
+					<p>Hi this is where I should put some information about myself</p>
+				</div>
+			</div>		
```

Comment:
Code indentation slightly off, all children of a parent should start at the same indent, indent again only when adding a grand-child. Could replace some `<div>`s with semantic tags such as `<article>` and all the `<sections>` could go in a `<main>` to help SEO etc